pkgs/os-specific/linux/compsize/default.nix

```diff
@@ -13,18 +13,17 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ btrfs-progs ];
 
-  installPhase = ''
-    mkdir -p $out/bin
+  PREFIX = placeholder "out";
+
+  preInstall = ''
     mkdir -p $out/share/man/man8
```

Comment:
```suggestion
  preInstall = ''
    mkdir -p $out/share/man/man8
    export PREFIX="${placeholder "out"}";
```

Reply:
Why?

Reply:
Defining ENVs directly does not make it clear in which phase it is needed and also can cause all sorts of escaping and formatting issues.

Reply:
That's fair enough, however setting the variables your proposed way makes it a bash specific implementation detail which is not nice either. Passing it as an `installFlag` now instead.